backend/note_taking/serializers/block_serializer.py

```
@@ -20,4 +20,4 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Block
-        fields = ["id", "block_type", "block_content", "note", "note_id", "block_order"]
\ No newline at end of file
+        fields = ["id", "type", "content", "note", "note_id", "position"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)